build/rpm/dependencies.yaml

```diff
@@ -51,6 +51,8 @@
 
 'augeasproviders_mounttab':
```

Comment:
The addition of another dependency requires the release qualifier to be bumped up.  It has to be larger than the previous one.